buildSrc/src/main/kotlin/ort-application-conventions.gradle.kts

```diff
@@ -177,3 +177,7 @@ tasks.named<CreateStartScripts>("startScripts") {
         )
     }
 }
+
+tasks.withType<Jar>().configureEach {
+    manifest.attributes["Main-Class"] = application.mainClass
```

Comment:
Is it actually supported to run ORT in this way (apart from this entry)? Or would ORT run into further issues ?

Reply:
See [above](https://github.com/oss-review-toolkit/ort/pull/7718#issuecomment-1771646498).